api-gateway/src/main/resources/static/scripts/auth/update-role-form/role-update.service.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+angular.module('userModule').service('UserService', [function() {
+    this.getAvailableRoles = function() {
+        return ['ADMIN', 'VET', 'OWNER'];
```

Comment:
Why are these values not fetched from the database, what happens if a new role is added ? For example I added the inventory manager ?